### ISSUE TITLE
Stop Loss as a Sell Strategy

### ISSUE THREAD

**Issue by RobinopdeBeek** [taniman/profit-trailer-enhancements#561]
Would be nice to have stop loss as a sell strategy, so you can combine it with other indicators to do a quick check if there is bounce back potential. With the new formula capability you can set it up like  (gain && ...) || (StopLoss && ...)

**Comment by taniman** [comment]
With this you mean you want to make the bot stoploss if certain conditions apply?

**Comment by RobinopdeBeek** [comment]
@taniman Yes exactly, for instance if the RSI < 30 or LOWBB < 10 the coin still has some bounce back potential, and should not stoploss yet. So you would get (Stoploss = -8 && RSI > 30 && LOWBB > 10). 
Or do you think that wouldn't work?

**Comment by Masterofcoinak** [comment]
Thats also part of my idea #568. But I want to have to the possibility of selling part of the bag (last DCA step).

**Comment by taniman** [comment]
You are aware you could just use RSI and LOWBB to sell right? Or do you specifocally only want it to trigger ones P% is below -8

That would explain the need for a STOPLOSS strategy
 Else youe formula is already possible

**Comment by thysbotwhisper** [comment]
This is already possible. The following would sell between -99 and -8, with RSI above 30. If you want to combine with with normal sell settings you can use an OR formula.

DEFAULT_A_sell_strategy = GAIN
DEFAULT_A_sell_value = -99
DEFAULT_A_sell_value_limit = -8

DEFAULT_B_sell_strategy = RSI
DEFAULT_B_sell_value = 30

**Comment by Masterofcoinak** [comment]
@thysbotwhisper sure it's possible. I think meanwhile it's not even necessary to use GAIN, 
The idea is, as @taniman has written, to start it below a certain value.  
I would love it, when there is another trigger then percentage. Make a balance related one.

**Comment by RobinopdeBeek** [comment]
@thysbotwhisper yeah good one! That would serve the same purpose. 
Still sounds a bit weird to use "gain" as a stoploss tho

**Comment by Masterofcoinak** [comment]
I think, the easiest and fastest integration into PT would be the possibility of setting LOSS as sell strategy. Like this it would be possible to set a formula. One with GAIN and 1 with LOSS

**Comment by RobinopdeBeek** [comment]
@thysbotwhisper One issue with this solution. It doesn't work if you also want to use trailing profit with your "positive" sell strategy. 
The solution that @Masterofcoinak would be an elegant one that would solve it.

**Comment by RobinopdeBeek** [comment]
Oh wait, the LOSS strategy looks at the 24h percentage drop. So I guess you would still need a seperate STOPLOSS sell strategy

**Comment by taniman** [comment]
we have PROFITPERCENTAGE strategy should do the trick
